pontoon/messaging/static/js/messaging.js

```diff
@@ -162,13 +163,87 @@ $(function () {
     $('#review .message-type .transactional').toggle(isTransactional);
   }
 
+  function loadPanelContent(path) {
+    if (inProgress) {
+      inProgress.abort();
+    }
```

Comment:
Is `inProgress` ever set back to `false`, or is `loadPanelContent()` only meant to be callable once?